Makefile

```diff
@@ -229,10 +229,6 @@ integration_pr: $(MINISHIFT_BINARY)
 fmt: ## Format source using gofmt
 	@gofmt -l -s -w $(SOURCE_DIRS)
 
-.PHONY: vet
-vet: ## Vet the source using 'go vet'
-	@go vet $(shell $(PACKAGES))
-
```

Comment:
We don't require `vet` as a separate target now since with golang 1.10 it by default run as part of `go test` 

- https://groups.google.com/forum/#!topic/golang-dev/qfa3mHN4ZPA

```
$ go test -h
[..]
	-vet list
	    Configure the invocation of "go vet" during "go test"
	    to use the comma-separated list of vet checks.
	    If list is empty, "go test" runs "go vet" with a curated list of
	    checks believed to be always worth addressing.
	    If list is "off", "go test" does not run "go vet" at all.
```
```